pkgs/tools/admin/pulumi/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "pulumi-${version}";
+  version = "0.16.2";
+
+  pulumiArchPackage = {
```

Comment:
Putting this in the `mkDerivation` will mean potential useless rebuilds, eg. rebuilding `x86_64-linux` even if only `x86_64-darwin` changes.